zebrad/src/config.rs

```diff
@@ -43,4 +43,8 @@ pub struct ZebradConfig {
     #[serde(skip_serializing_if = "zebra_rpc::config::mining::Config::skip_getblocktemplate")]
     /// Mining configuration
     pub mining: zebra_rpc::config::mining::Config,
+
+    #[cfg(feature = "zebra-scan")]
+    /// Scanner configuration
+    pub scan: zebra_scan::config::Config,
```

Comment:
Optional: We've had some user and developer confusion between shielded scanning (which is this feature), and transparent scanning (which we already do).

Would it help to rename the config?
```suggestion
    #[cfg(feature = "zebra-scan")]
    /// Scanner configuration
    pub shielded_scan: zebra_scan::config::Config,
```